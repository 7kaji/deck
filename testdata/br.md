# Hello<br>World

Hello<br>World
