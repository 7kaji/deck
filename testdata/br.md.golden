[
  {
    "layout": "",
    "titles": [
      "Hello\nWorld"
    ],
    "bodies": [
      {
        "paragraphs": [
          {
            "fragments": [
              {
                "value": "Hello"
              },
              {
                "value": "\n"
              },
              {
                "value": "World"
              }
            ]
          }
        ]
      }
    ]
  }
]